framework/base/Security.php

```diff
@@ -30,6 +30,11 @@
  */
 class Security extends Component
 {
+    const DEV_URANDOM = '/dev/urandom';
```

Comment:
i know. but i have been chided often for using "magic numbers". they are considered bad practice.

http://refactoring.com/catalog/replaceMagicNumberWithSymbolicConstant.html

for example, in this file we '/dev/urandom' four times. with a constant, the static inspections ensure we don't have a typo in any of them.